modules/rostests/apitests/crt/dll_startup.cpp

```diff
@@ -8,6 +8,10 @@
 #include <apitest.h>
 #include "dll_startup.h"
 
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wuninitialized"
```

Comment:
This is probably about `m_uninit`?
Would be nice to indicate it with a comment.